docs/developers/contributing/performance/benchmarks.md

```diff
@@ -102,6 +102,22 @@ of time since building napari can be a time consuming task:
 asv run -E existing -b ViewImageSuite
 ```
 
+## Debugging a benchmark
+
+To simplify run a benchmark in a debugger, you can use the command:
+
+```bash
+python -m pdb napari/_benchmarks/benchmark_file.py BenchmarkClass.benchmark_method
+```
+
+For example, to debug the ``time_set_view_slice`` method in the ``Image2DSuite``
+benchmark, you can run:
```

Comment:
```suggestion
To run a benchmark in a debugger, you can use the command:

```bash
python -m pdb napari/_benchmarks/benchmark_file.py BenchmarkClass.benchmark_method
```

For example, to debug the ``time_set_view_slice`` method in the ``Image2DSuite``
benchmark, you can run:
```

Reply:
Looks like you should edit the comment and use four tics instead of three.